frontend/src/app/main/data/workspace/drawing.cljs

```diff
@@ -27,6 +27,8 @@
   (ptk/reify ::select-for-drawing
     ptk/UpdateEvent
     (update [_ state]
+      (when (= tool :comments)
+        (update state :workspace-layout disj :document-history))
```

Comment:
This code does nothing, I mean, it returns a new state but this state is ignored.